obal/data/playbooks/mock/mock.yaml

```diff
@@ -0,0 +1,17 @@
+---
+- hosts:
+    - packages
+  serial: 1
+  gather_facts: no
+  roles:
+    - build_srpm
+
+- hosts:
+    - packages
+  serial: 1
+  gather_facts: no
+  tasks:
+    - shell: "mock --recurse --chain -r {{ mock_config }} --localrepo {{ inventory_dir }}/mock_builds SRPMs/*"
```

Comment:
In build_srpm we have a variable for this so we can change it:

https://github.com/theforeman/obal/blob/2f21ce6999b337bb1efd1f8c6cc12287242510d4/obal/data/roles/build_srpm/defaults/main.yaml#L3

Since these are usually the same, it might be nice to also make it a variable here.

Maybe the same goes for `mock_builds`. We should include those in our `.gitignore` and recommend other projects to do the same.